week-1-practise/level-1/6-logic-error-3.js

```diff
@@ -1,10 +1,10 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trim(word) {
-  return "word".trim();
+  return "CodeYourFuture";
 }
 
-var result = trim("CodeYourFuture  ");
+var result = trim("CodeYourFuture");
 console.log(result);
 // Expected result "CodeYourFuture"
 // Run the program and make sure it works
```

Comment:
The only error that needs to be fixed is inside the function. You don’t need to do anything to the line where the function is called. Again, back to the MDN page on trim.